spynnaker/pyNN/models/pynn_projection_common.py

```diff
@@ -267,6 +267,10 @@ def _get_synaptic_data(self, as_list, data_to_get):
         connection_holder.finish()
         return connection_holder
 
+    def _clear_cache(self):
+        post_vertex = self._projection_edge.post_vertex
+        post_vertex.clear_connection_cache()
```

Comment:
fixed, and added the call to the interface as well. verified that only APV uses it, but better safe than sorry.